components/style/properties/gecko.mako.rs

```diff
@@ -974,6 +974,25 @@ fn static_assert() {
         self.gecko.mFont.synthesis = other.gecko.mFont.synthesis;
     }
 
+    pub fn set_font_size_adjust(&mut self, v: longhands::font_size_adjust::computed_value::T) {
+        use properties::longhands::font_size_adjust::computed_value::T;
+        match v {
+            T::None => self.gecko.mFont.sizeAdjust = -1.0 as f32,
+            T::Number(n) => self.gecko.mFont.sizeAdjust = n.0 as f32,
+        }
+    }
+
+    pub fn copy_font_size_adjust_from(&mut self, other: &Self) {
+        self.gecko.mFont.sizeAdjust = other.gecko.mFont.sizeAdjust;
+    }
+
+    pub fn clone_font_size_adjust(&self) -> longhands::font_size_adjust::computed_value::T {
```

Comment:
`clone_` isn't related to `Clone`, it's usually used for animation. From Servo's POV the style structs store servo computed values, so "cloning" one out in stylo is a more complex operation than a regular clone.